docker/Dockerfile

```diff
@@ -308,7 +311,7 @@ USER nautobot
 
 WORKDIR /opt/nautobot
 
-RUN nautobot-server init
+COPY --from=final-dev /opt/nautobot/nautobot_config.py /opt/nautobot/nautobot_config.py
```

Comment:
Got an issue `cannot create /opt/nautobot/nautobot_config.py: Permission denied`
Running `cat /opt/nautobot/nautobot_config.append.py >> /opt/nautobot/nautobot_config.py` while building homemade Dockerfile using Nautobot docker image as base

I think adding `--chown=nautobot` should solve the issue